ivy/functional/ivy/device.py

```diff
@@ -305,8 +308,30 @@ def print_all_ivy_arrays_on_dev(
     ----------
     device
         The device on which to print the arrays
+    
     attr_only
         Whether or not to only print the `shape` and `dtype` attributes of the array
+    
+    Examples
+    --------
+
+    With attr_only = True
+
+    >>> x = ivy.array([1,0,2], [3,2,1])
```

Comment:
The array constructor does not accept two arrays as input. You should change the input to be a single array with 2 dims instead as `x = ivy.array([[1,0,2], [3,2,1]])`. The same applies to the other example. Thanks!

Reply:
Both comments should now be fixed.